mainnet/prysm/prysm.yaml

```diff
@@ -16,4 +16,4 @@ minimum-peers-per-subnet: 0
 block-batch-limit: 512
 block-batch-limit-burst-factor: 10
 
```

Comment:
We should set all addresses to `localhost` (all `'0.0.0.0'` except `p2p-host-ip`)